basic/exercise10/index.html

```diff
@@ -3,10 +3,18 @@
 <head>
```

Comment:
File is not formatted - when I choose file formatting, then some extra empty lines are added